### PR TITLE
Update DIVD-2021-00022.md

### PR DESCRIPTION
**Pull request by Lennaert89**
closing this case

### CODE REVIEW

**Review by Ralphhorn — COMMENTED** [review]
lgtm

File: _cases/2021/DIVD-2021-00022.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ timeline:
 - start: 2021-08-30   
   end:
   event: DIVD releases [scan script](https://github.com/DIVD-NL/ProxyOracleNSE) for CVE-2021-31195
+- start: 2022-10-10
+  end:
+  event: Closing this case
 ips: 0
```
Comment (MrSeccubus) [inline _cases/2021/DIVD-2021-00022.md]: IIRC (And as case lead I should), we had quite a bit more then 0 affected IPs.

Comment (MrSeccubus) [reply _cases/2021/DIVD-2021-00022.md]: Fixed it.

File: _cases/2021/DIVD-2021-00022.md [hunk 2]
```diff
@@ -63,6 +66,8 @@ If you are having trouble getting your Exchange server patched, we suggest that
 
 We have made an [NSE script for ProxyOracle](https://github.com/DIVD-NL/ProxyOracleNSE) available via out GitHub account. We are going to scan internet facing Exchange servers for ProxyOracle and ProxyShell (via [Kevin beaumont's script](https://github.com/GossiTheDog/scanning/blob/main/http-vuln-exchange-proxyshell.nse))
 
+For us this case has ended, anyone still vulnerable to these issues over a year after patches were made available will be notified of this together with [case 2022-00054](https://csirt.divd.nl/cases/DIVD-2022-00054/).
```
Comment (MrSeccubus) [inline _cases/2021/DIVD-2021-00022.md:69]:
```suggestion
For us this case has ended, anyone still vulnerable to these issues over a year after patches were made available will be notified of this together with [case 2022-00054](/cases/DIVD-2022-00054/).
```
Relative url's please.

**Review by MrSeccubus — CHANGES_REQUESTED** [review]

**Pull request merged**